tests/infer/from_master/master_inferrers.py

```diff
@@ -0,0 +1,87 @@
+"""Supplementary inferrers for testing.
```

Comment:
Could you keep `from_master` in a different branch? I don't see a purpose in merging these in.

Reply:
Done!